### PR TITLE
Added info for Terri Oda in the speakers section

### PR DESCRIPTION
**Pull request by aditya-konarde**
Added the information and photo in the speakers section. This pull request will fix issue #22 
Please review the PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-  
+
```
Comment (SaptakS) [inline index.html:5]: Unnecessary new line

Comment (aditya-konarde) [reply index.html:5]: The line was already there, and contained white space. It seems that Atom removed the trailing white space all over the file automatically. I feel this is very minor and can be fixed later. Let me know if it still has to be fixed.

Comment (aditya-konarde) [reply index.html:5]: @SaptakS going through the other files, I see that there are whitespaces and unnecessary new lines at many places. Thanks for pointing this out. Shall we add an issue for this or is it minor? I can work on removing them if there's a need.

**Comment by sayanchowdhury** [comment]
The picture is not from the link specified in the issue.

**Pull request closed without merging**